app/frontend/Calendar/EditEvent/EditEvent.svelte

```diff
@@ -350,6 +350,7 @@
   }
   .left-pane {
     flex: 2 0 0;
+    width: 0px;
```

Comment:
What is this?

Reply:
This tells layout not to take the width of the content into account when determining the width of the left pane. By default this will cause the editor text to wrap, so I added the min-width on the editor if you'd prefer it to scroll.